src/main/scala/com/ironcorelabs/DeviceContext.scala

```diff
@@ -7,21 +7,40 @@ import com.ironcorelabs.{sdk => jsdk}
 /**
  * Account's device context. Needed to initialize the SDK with a set of device keys.
  *
- * @param id ID of a user
+ * @param deviceId ID of a device
+ * @param userId ID of a user
  * @param segmentId ID of a segment
- * @param privateDeviceKey asymmetric private key
- * @param deviceSigningKeyPair signing keypair specific to a device
+ * @param devicePrivateKey asymmetric private key
+ * @param signingPrivateKey signing keypair specific to a device
```

Comment:
If it is a keypair, shouldn't it be `signingKeyPair`?

Reply:
I changed the name to match our new standardized serialization/deserialization as documented in https://github.com/ironcorelabs/ironoxide/issues/25 
The comment there with the signingPrivateKey is: `// “expanded private key” (both pub/priv)`